.scaffold/tests/bats/ahoy.bats

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bats
+
+load _helper
+
+export BATS_FIXTURE_EXPORT_CODEBASE_ENABLED=1
+
+@test "ahoy assemble" {
+  run ahoy assemble
+  assert_success
+
+  assert_output_contains "ASSEMBLE COMPLETE"
+  assert_dir_exists "${BUILD_DIR}/build/vendor"
+  assert_file_exists "${BUILD_DIR}/build/composer.json"
+  assert_file_exists "${BUILD_DIR}/build/composer.lock"
+
+  ahoy reset
```

Comment:
remove this as we are not testing reset

if you need to stop the PHP processes or do other cleanups - create a bats helper and call it here. the contents of that helper could be identical to what `ahoy stop` does. This is to make sure that changes to `ahoy stop` does not break tests that are not testing it.

```
stop_server(){
  killall -9 php >/dev/null 2>&1 || true
  sleep 1
}
```